### ISSUE TITLE
MDTextField not has rightView/leftView :(

### ISSUE THREAD

**Issue by sphairo** [fpt-software/Material-Controls-For-iOS#83]
Me gustaría poder usar esta propiedad que tenia el UITextField ahora en sus nuevos componentes MDTextField.

**Comment by ketan-parmar** [comment]
Yeah! this is very important property which should be in MDTextField, but unfortunately currently not available!
